lib/vfscore/exportsyms.uk

```diff
@@ -81,16 +85,23 @@ uk_syscall_e_lstat
 uk_syscall_r_lstat
 lchown
 openat
+uk_syscall_e_openat
+uk_syscall_r_openat
 opendir
 readdir
 readdir_r
 readdir64
 closedir
 pread
```

Comment:
Shouldn't there also be a `pread64` added (similarly to how `pwrite64` is registered)?